src/app/questions/browse/page.tsx

```diff
@@ -53,7 +53,15 @@ const QuestionsPage: React.FC = () => {
           response.data.questions.forEach((question) => {
             uniqueQuestionsMap.set(question.question_id.toString(), question)
           })
-          const uniqueQuestions = Array.from(uniqueQuestionsMap.values())
+          let uniqueQuestions = Array.from(uniqueQuestionsMap.values())
```

Comment:
we should probably move this to the backend for consistency with our current pagination/sorting

Reply:
but we can also fix this later